CONTRIBUTING.md

```diff
@@ -20,3 +20,4 @@ Please be civil when taking part in discussions.
 For pull requests, please follow the [GitHub flow](https://docs.github.com/en/github/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request-from-a-fork): create a fork of the repository, create your own branch, do commits, push changes to your branch, do a pull request (PR).
 The destination branch of pull requests is the default `master` branch.
 
+For code contributions, please respect the [Linux Kernel coding style](https://www.kernel.org/doc/html/v4.10/process/coding-style.html). To make this process easier, download [checkpatch.pl](https://github.com/systems-cs-pub-ro/iocla/blob/master/scripts/checkpatch.pl) and run [this command](https://github.com/systems-cs-pub-ro/iocla/blob/master/.github/workflows/main.yml#L12), it will show you all style errors. Alternatively, there's a [VS Code extension](https://marketplace.visualstudio.com/items?itemName=idanp.checkpatch) that integrates the patch into its editor (follow the instructions on the extension page).
```

Comment:
Place each sentence in this paragraph on a separate line. This way it'll be easier to review. You should add [this paragraph](https://github.com/open-education-hub/operating-systems/blob/master/CONTRIBUTING.md#content-writing-style) to `CONTRIBUTING.md` as well.